File: src/main/kotlin/com/stannis/dataModel/complexStatementTypes/ComplexFinalTranslation.kt
package com.stannis.dataModel.complexStatementTypes

import com.stannis.dataModel.Statement
import com.stannis.parser.sln.VcxprojStructure

data class ComplexFinalTranslation(
    override val `$type`: String? = "ComplexFinalTranslation",
    val listOfHeaderFiles: ArrayList<TranslationWithPath>?,
    val listOfCppFiles: ArrayList<TranslationWithPath>?,
    val vcxprojStructure: VcxprojStructure
): Statement

File: Interpreter/Interpreter/Models/ComplexFinalTranslation.cs
using System.Collections.Generic;
using Interpreter.Models.complexStatementTypes;

namespace Interpreter.Models{
    public class ComplexFinalTranslation : IStatement
    {
            public VcxprojStructure vcxprojStructure { get; set; }

            public IList<TranslationWithPath> listOfHeaderFiles { get; set; } = new List<TranslationWithPath>();

            public IList<TranslationWithPath> listOfCppFiles { get; set; } = new List<TranslationWithPath>();
    }
};